File: BaekJoon/Sorting/10989.py
import sys


N = int(sys.stdin.readline())

result = []

for _ in range(10000):
    result.append(0)

for _ in range(N):
    num = int(sys.stdin.readline())
    result[num-1]+=1

for i in range(10000):
    for _ in range(result[i]):
        print(i+1)
    

File: BaekJoonSwift/BaekJoonSwift/Sorting/10989.swift
import Foundation

let n = Int(readLine()!)!
var result = [Int]()
var str = ""
for _ in 0...10000 {
    result.append(0)
}

for _ in 0..<n {
    let temp = Int(readLine()!)!
    result[temp] += 1
}

for i in 1...10000 {
    for _ in 0..<result[i] {
        str += "\(i)\n"
    }
}

print(str)
